src/main/java/duke/task/ToDos.java

```diff
@@ -0,0 +1,29 @@
+package duke.task;
+
+import duke.exception.DukeException;
+
+public class ToDos extends Task {
+
+    public ToDos(String description) {
+        super(description);
+    }
+
+    public static Task parse(String taskInfo) throws DukeException {
+        if (taskInfo.isBlank()) {
+            // the string is empty or contains only white space
```

Comment:
Try to keep the language of your comments consistent. In other files, you used comments to describe what is happening but now you are using the comments to specify the condition. Stick to one and use that style throughout your codebase.